### ISSUE TITLE
Look into why the merge code was splitting

### ISSUE THREAD

**Issue by keith-turner** [apache/accumulo#3605]
The tablet merge code was adding splits to the table at the merge boundaries and calling the tablet server to do the split.  This code is there in the elasticity branch but does not function after changes like #3604.  Need to determine why the merge code was doing this and if was tightly coupled to chop merges and would even be needed after the changes in #1327.  If the split at merge boundry is still needed even with #1327, then will need to have the merge code call new split code in the manager.

**Comment by keith-turner** [comment]
There is a method in TabletGroupWatcher named sendSplitRequest() that would call the split RPC on tsever.  This method is used by the merge code.   #3604 made this method throw UnsupportedOperationException, which motivated opening this follow on issue.

**Comment by cshannon** [comment]
I can look into why the merge code is splitting later this week as I'm already in that code anyways with #1327

**Comment by cshannon** [comment]
I looked into this and the split request is only done for when a range is deleted from a table and not during a merge. It's kind of interesting how things are designed as merging a range and deleting a range code is coupled together and the MergeInfo object defines the [Operation](https://github.com/apache/accumulo/blob/4ec6dd2054d6a588dfc5a9a5f139d32799c90e83/server/base/src/main/java/org/apache/accumulo/server/manager/state/MergeInfo.java#L36) type

Inside of the split request code you can see that if it's not a delete (so a merge) it just [returns](https://github.com/apache/accumulo/blob/4ec6dd2054d6a588dfc5a9a5f139d32799c90e83/server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java#L567). 

I didn't look into the deletes yet but based off on this I will also need to update #3640 to handle metadata updates for range deletion. If we can update the code to do proper fencing of the files then maybe we can avoid having to do splits and the Elasticity branch wouldn't need to worry about it.

**Comment by keith-turner** [comment]
> If we can update the code to do proper fencing of the files then maybe we can avoid having to do splits and the Elasticity branch wouldn't need to worry about it.

That would be nice, would simplify the algorithm.

**Comment by keith-turner** [comment]
Can probably close this issue after #3761 is merged into elasticity.
